WordPress/Classes/Utility/App Configuration/AppConfiguration.swift

```diff
@@ -9,11 +9,4 @@ import Foundation
 @objc class AppConfiguration: NSObject {
     @objc static let isJetpack: Bool = false
     @objc static let isWordPress: Bool = true
```

Comment:
When https://github.com/wordpress-mobile/WordPress-iOS/pull/24214 is merged, I'll remove these two remaning fields as well. The plan is to replace it with an enum and put it in `BuildSettingsKit` together with other build settings.